docs/resources/shadow.md.erb

```diff
@@ -238,7 +238,7 @@ An integer reflecting the number of days that must pass before a user who has no
 
 ### expiry_date
 
-An integer reflecting the number of days since Jan 1, 1970 on which the user was disabled. The value `nil` means the account was disabled because of an expired password.
+An integer reflecting the number of days since Jan 1, 1970 on which the user was disabled. The value `nil` means the account has not been disabled.
```

Comment:
"has not been disabled" is both passive & subjunctive. Can you find a solution using the present tense and the active voice.